--- itcl3.2.1/tcl.m4.orig	2001-06-22 06:41:05.000000000 +0200
+++ itcl3.2.1/tcl.m4	2004-01-31 21:50:28.063410149 +0100
@@ -1859,12 +1859,14 @@
     # wasn't specified since a search in the unspecified directory will
     # fail (NONE/lib)
 
-    if test x"${exec_prefix}" != x"NONE" ; then
+    if test x"${libdir}" != x"NONE" ; then
+	sc_lib_name_dir="${libdir}"
+    elif test x"${exec_prefix}" != x"NONE" ; then
 	sc_lib_name_dir="${exec_prefix}/lib"
     elif test x"${prefix}" != "NONE" ; then
 	sc_lib_name_dir="${prefix}/lib"
     else
-	eval "sc_lib_name_dir=${libdir}"
+	sc_lib_name_dir=
     fi
 
     if test x"$2" != x ; then
--- itcl3.2.1/itk/configure.in.orig	2001-08-13 18:14:29.000000000 +0200
+++ itcl3.2.1/itk/configure.in	2004-01-31 22:04:54.025972660 +0100
@@ -173,7 +173,9 @@
 #--------------------------------------------------------------------
 
 AC_MSG_CHECKING(for name if Itcl stub library)
-AC_CACHE_VAL(ac_cv_itclstub_LIB_FILE, ac_cv_itclstub_LIB_FILE="NONE")
+itclstub_LIB_FILE=NONE
+eval "`grep ^itclstub_LIB_FILE ../itcl/config.log`"
+AC_CACHE_VAL(ac_cv_itclstub_LIB_FILE, ac_cv_itclstub_LIB_FILE="$itclstub_LIB_FILE")
 if test x"${ac_cv_itclstub_LIB_FILE}" = x"NONE" ; then
     AC_MSG_ERROR(Can't find name of Itcl stub library.  How did this happen?)
 else
