--- itcl4.2.3/Makefile.in.orig	2022-11-04 15:02:05.000000000 +0100
+++ itcl4.2.3/Makefile.in	2024-07-05 21:31:56.048730701 +0200
@@ -290,8 +290,7 @@ genstubs: $(srcdir)/tools/genStubs.tcl $
 
 $(PKG_LIB_FILE): $(PKG_OBJECTS)
 	-rm -f $(PKG_LIB_FILE)
-	${MAKE_LIB}
-	$(RANLIB) $(PKG_LIB_FILE)
+	${MAKE_LIB} -Wl,-soname,$(PKG_LIB_FILE)
 
 $(PKG_STUB_LIB_FILE): $(PKG_STUB_OBJECTS)
 	-rm -f $(PKG_STUB_LIB_FILE)
