--- itcl/Makefile.in.orig	2003-12-31 17:35:30.000000000 +0100
+++ itcl/Makefile.in	2003-12-31 18:27:33.782467160 +0100
@@ -55,7 +55,7 @@
 # library that we are building.
 #========================================================================
 
-PKG_LIB_FILE	= @PKG_LIB_FILE@
+PKG_LIB_FILE	= @PKG_LIB_FILE@.0.0
 PKG_STUB_LIB_FILE = @PKG_STUB_LIB_FILE@
 
 lib_BINARIES	= $(PKG_LIB_FILE) $(PKG_STUB_LIB_FILE)
@@ -230,8 +230,8 @@
 
 $(PKG_LIB_FILE): $(PKG_OBJECTS)
 	-rm -f $(PKG_LIB_FILE)
-	${MAKE_LIB}
-	$(RANLIB) $(PKG_LIB_FILE)
+	${MAKE_LIB} -Wl,-soname,libitcl$(VERSION).so.0
+	-ln -sf $(PKG_LIB_FILE) libitcl$(VERSION).so.0
 
 $(PKG_STUB_LIB_FILE): $(PKG_STUB_OBJECTS)
 	-rm -f $(PKG_STUB_LIB_FILE)
