--- itcl/Makefile.in.orig	2003-12-31 17:35:30.000000000 +0100
+++ itcl/Makefile.in	2003-12-31 18:27:33.782467160 +0100
@@ -55,7 +55,7 @@
 # library that we are building.
 #========================================================================
 
-PKG_LIB_FILE	= @PKG_LIB_FILE@
+PKG_LIB_FILE	= @PKG_LIB_FILE@.0.0
 PKG_STUB_LIB_FILE = @PKG_STUB_LIB_FILE@
 
 lib_BINARIES	= $(PKG_LIB_FILE) $(PKG_STUB_LIB_FILE)
@@ -230,8 +230,8 @@
 
 $(PKG_LIB_FILE): $(PKG_OBJECTS)
 	-rm -f $(PKG_LIB_FILE)
-	${MAKE_LIB}
-	$(RANLIB) $(PKG_LIB_FILE)
+	${MAKE_LIB} -Wl,-soname,libitcl$(PACKAGE_VERSION).so.0
+	-ln -sf $(PKG_LIB_FILE) libitcl$(PACKAGE_VERSION).so.0
 
 $(PKG_STUB_LIB_FILE): $(PKG_STUB_OBJECTS)
 	-rm -f $(PKG_STUB_LIB_FILE)
--- itcl3.4.2/pkgIndex.tcl.in.orig	2011-07-15 18:34:43.000000000 +0200
+++ itcl3.4.2/pkgIndex.tcl.in	2011-12-02 17:07:08.483164656 +0100
@@ -1,4 +1,4 @@
 # Tcl package index file, version 1.0
 
 if {![package vsatisfies [package provide Tcl] @TCL_VERSION@]} {return}
-package ifneeded Itcl @PACKAGE_VERSION@ [list load [file join $dir "@PKG_LIB_FILE@"] Itcl]
+package ifneeded Itcl @PACKAGE_VERSION@ [list load "@libdir@/@PKG_LIB_FILE@.0" Itcl]
