--- itcl-3.3/iwidgets4.0.1/configure.in~	2001-11-29 21:51:34.000000000 +0100
+++ itcl-3.3/iwidgets4.0.1/configure.in	2008-04-02 14:40:20.000000000 +0200
@@ -63,12 +63,10 @@
 ITCL_LIB_DIR=""
 if test -r $itcl_search/itcl/library/itcl.tcl; then
     if test -r $itcl_search/itk/library/itk.tcl; then
-        if test -r $itcl_search/config/mkinstalldirs; then
             ITCL_LIB_DIR=$itcl_search/itcl/library
             ITCL_SRC_DIR=$itcl_search/itcl
             ITK_LIB_DIR=$itcl_search/itk/library
             ITK_SRC_DIR=$itcl_search/itk
-        fi
     fi
 fi
 
